pkgs/tools/networking/uwimap/default.nix

```diff
@@ -47,7 +48,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.washington.edu/imap/";
     description = "UW IMAP toolkit - IMAP-supporting software developed by the UW";
     license = lib.licenses.asl20;
```

Comment:
Should probably use `meta = with lib; {` above so that we dont have to specify `lib` twice here.